pyanaconda/bootloader.py

```diff
@@ -1799,7 +1798,8 @@ class EFIGRUB1(EFIBase, GRUB):
     _efi_binary = "\\grub.efi"
 
     def __init__(self):
-        super(EFIGRUB1, self).__init__()
+        EFIBase.__init__(self)
+        GRUB.__init__(self)
```

Comment:
Please, call here just `super().__init__()`.

Classes in this file handle multiple inheritance in a different way. They let python to decide the order of the calls. It would be nice to replace it with manual calls, but it will require a careful analysis and a lot of testing to make sure that the classes are initialized correctly.